src/pharmpy/modeling/odes.py

```diff
@@ -1469,3 +1471,93 @@ def set_ode_solver(model, solver):
     for step in model.estimation_steps:
         step.solver = solver
     return model
+
+
+def find_clearance_parameters(model):
+    """Find clearance parameters in model
+
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model
+
+    Return
+    ------
+    list
+        A list of clearance parameters
+
+    Examples
+    --------
+     >>> from pharmpy.modeling import *
+     >>> model = load_example_model("pheno")
+     >>> find_clearance_parameters(model)
+
+    """
+    cls = []
+    sset = model.statements
+    rate_list = _find_rate(sset)
+    for rate in rate_list:
+        clearance = rate.as_numer_denom()[0]
+        if clearance.is_Symbol:
+            clearance = _find_real_symbol(sset, clearance)
+            if clearance not in cls:
+                cls.append(clearance)
+    return cls
+
+
+def find_volume_parameters(model):
+    """Find volume parameters in model
+
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model
+
+    Return
+    ------
+    list
+        A list of volume parameters
+
+    Examples
+    --------
+     >>> from pharmpy.modeling import *
+     >>> model = load_example_model("pheno")
+     >>> find_volume_parameters(model)
```

Comment:
This will not pass the doctest

Reply:
You need to add the function to the `__init__.py` then do `tox -e docs,doctest`. It should fail since no output from the function call is given.